.env

```diff
@@ -0,0 +1 @@
+WEBSITE_API_URL=https://json-server-beta-six.vercel.app
```

Comment:
Deal with .env file is important. Please refer to my previous recording video about the environment variables.

Reply:
renewed .env file and ignore uploading